backend/src/controller/SubscriptionController.js

```diff
@@ -0,0 +1,55 @@
+export class SubscriptionController {
+    constructor (db) {
+        this.db = db
+    }
+
+    isValidSaveRequest (req, res) {
+        // Check the request body has at least an endpoint.
+        if (!req.body || !req.body.endpoint) {
+            // Not a valid subscription.
+            res.status(400)
+            res.setHeader('Content-Type', 'application/json')
+            res.send(
```

Comment:
Wanneer je een object verzend word deze automatisch omgezet naar json en word de content-type header toegevoegt. Is netter dan dit zelf overal doen